internal/storage/bucket_handle.go

```diff
@@ -233,10 +232,10 @@ func (bh *bucketHandle) CreateObjectChunkWriter(ctx context.Context, req *gcs.Cr
 	wc.ProgressFunc = callBack
 	// All objects in zonal buckets must be appendable.
 	wc.Append = bh.BucketType().Zonal
-	// Objects in zonal buckets should not finalized.
-	if bh.BucketType().Zonal {
-		wc.FinalizeOnClose = false
-	}
+	// Objects in zonal buckets should not be finalized by default. Finalize them if finalizeFileForRapid is set to true.
+	// When writer.Append is false,then this parameter is anyways ignored.
```

Comment:
Do we have test to catch, in case `When writer.Append is false,then this parameter is anyways ignored` behavior changed?